canonical-websites/www.ubuntu.com
# 3848: Add copydocs metadata to all pages

## Done

- Added copydocs metadata to almost every page on Ubuntu, just missing a few legal pages I think
- Tidied up the metadata in the templates a little bit, so it goes Title -> Description -> Copydoc link

## QA

- Check out this feature branch
- Download the Ubuntu Copydoc extension from [here](https://github.com/canonical-webteam/ubuntu-copy-docs) (thanks @bartaz!) for either Firefox or Chrome
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Navigate to as many pages as you can be bothered, and check that the copydoc link at the bottom of the page is correct

## Issue / Card

Fixes #3848
